tests/integration/targets/meraki_management_interface/tasks/main.yml

```diff
@@ -3,182 +3,199 @@
-      auth_key: '{{auth_key}}'
+  - name: 'Create test network {{ net_name }}'
+    cisco.meraki.meraki_network:
+      auth_key: '{{ auth_key }}'
       state: present
-      org_id: '{{test_org_id}}'
-      net_name: '{{net_name}}'
+      org_id: '{{ test_org_id }}'
+      net_name: '{{ net_name }}'
       type: appliance
     delegate_to: localhost
     register: net
 
-  - set_fact:
-      net_id: '{{net.data.id}}'
+  - name: Set fact net_id
+    ansible.builtin.set_fact:
+      net_id: '{{ net.data.id }}'
 
   - name: Test providing wan_enabled to an MS network
-    meraki_management_interface:
-      auth_key: '{{auth_key}}'
+    cisco.meraki.meraki_management_interface:
+      auth_key: '{{ auth_key }}'
       state: present
-      org_id: '{{test_org_id}}'
-      net_id: '{{test_switch_net_name}}'
-      serial: '{{serial_switch}}'
+      org_id: '{{ test_org_id }}'
+      net_id: '{{ test_switch_net_name }}'
+      serial: '{{ serial_switch }}'
       wan1:
         wan_enabled: enabled
         using_static_ip: false
     delegate_to: localhost
     register: ms_not_configured
 
-  - debug:
+  - name: Debug ms_not_configured
+    ansible.builtin.debug:
       var: ms_not_configured
 
-  - assert:
+  - name: Assert ms_not_configured
+    ansible.builtin.assert:
       that:
         - ms_not_configured.data is defined
 
   - name: Set management interface on switch
-    meraki_management_interface:
-      auth_key: '{{auth_key}}'
+    cisco.meraki.meraki_management_interface:
+      auth_key: '{{ auth_key }}'
       state: present
-      org_id: '{{test_org_id}}'
-      net_id: '{{test_switch_net_name}}'
-      serial: '{{serial_switch}}'
+      org_id: '{{ test_org_id }}'
+      net_id: '{{ test_switch_net_name }}'
+      serial: '{{ serial_switch }}'
       wan1:
-        using_static_ip: no
+        using_static_ip: false
         vlan: 3
     delegate_to: localhost
     register: set_switch_mgmt
 
-  - debug:
+  - name: Debug set_switch_mgmt
+    ansible.builtin.debug:
       var: set_switch_mgmt
 
-  - assert:
+  - name: Assert set_switch_mgmt has been defined
+    ansible.builtin.assert:
       that:
         - set_switch_mgmt.data is defined
 
   - name: Query non-MX network
-    meraki_management_interface:
-      auth_key: '{{auth_key}}'
+    cisco.meraki.meraki_management_interface:
+      auth_key: '{{ auth_key }}'
       state: query
-      org_id: '{{test_org_id}}'
-      net_id: '{{test_switch_net_name}}'
-      serial: '{{serial_switch}}'
+      org_id: '{{ test_org_id }}'
+      net_id: '{{ test_switch_net_name }}'
+      serial: '{{ serial_switch }}'
     delegate_to: localhost
-    register: non_mx_network
+    register: falsen_mx_network
 
-  - debug:
-      var: non_mx_network
+  - name: Debug falsen_mx_network
+    ansible.builtin.debug:
+      var: falsen_mx_network
 
-  - assert:
+  - name: Assert non_mx_network
+    ansible.builtin.assert:
       that:
         - non_mx_network.data is defined
```

Comment:
This needs to be updated as it's using the old variable name.

Reply:
My script changed the `no` to `false`. In that case it changed the `non_mx_network` to `false_mx_network` :D

Reply:
I pushed it ;)